### ISSUE TITLE
1.14.2_HD_U_F1_pre11 crashes

### ISSUE THREAD

**Issue by ws909** [sp614x/optifine#2481]
The game crashes when the progress bar reaches 100% on the loading screen. I have tested it 3 times. I have also downloaded and installed the preview versions of 1.14: 2, 5, 6, and for 1.14.2: 6, 7, 9. Looking back, I'm pretty confident I've not had any crashes with the other versions. I'm now back at using preview 9.

I'm attaching the log file of the second crash. I have removed my username, the shaderpack name, and the resource pack names. The shaderpack is for 1.12, and the main RP for 1.14.
[latest.log](https://github.com/sp614x/optifine/files/3287618/latest.log)

**Comment by ZachArtHunter** [comment]
That log doesn't have a crash message anywhere. You should upload the crash-report found in the minecraft/crash-reports folder.

**Comment by ws909** [comment]
The crash report in that folder hasn't been updated since 2018.

**Comment by ZachArtHunter** [comment]
When you crash, the launcher should pop up with a box that says GAME CRASH, near the bottom, right above the close button it should say [View crash report] if you click that it should open the crash report.

**Comment by ws909** [comment]
The "View crash report" button hasn't been present since 1.11 or 1.12.

**Comment by sp614x** [comment]
Can you repeat the same with pre10 and post the error log?

**Comment by sp614x** [comment]
Can you repeat the same with pre10 and post the error log?

**Comment by ws909** [comment]
I don't have the pre10 jar file. It's been replaced with pre11 on the website. When changing the web address on the mirror download, to pre10, I'm given an error message: "File not found" when attempting to download it. I assume you've removed pre10 from the server?

**Comment by sp614x** [comment]
Reuploaded pre10.

**Comment by ws909** [comment]
The game didn't crash with pre10, neither while loading the game itself, or by loading a world.

**Comment by sp614x** [comment]
Please post the log, I want to compare it with pre11.

**Comment by ws909** [comment]
[latest (pre10).log](https://github.com/sp614x/optifine/files/3287776/latest.pre10.log)

**Comment by AceKingPanda** [comment]
I've discovered that a specific resource pack that I have been using has been causing the crash for me. After disabling the resource pack, logging in, and re-enabling the pack, I have been able to generate an error log. The resource pack is Sphax PureBDCraft x128 and a WIP if that matters. I've also used a x64 and x32 resource pack without error. I'm clueless so I hope this helps. Attached will be the error log.
[hs_err_pid47044.log](https://github.com/sp614x/optifine/files/3287854/hs_err_pid47044.log)
